webapp/src/pages/JoinChatroomPage/JoinChatroomPage.tsx

```diff
@@ -78,64 +73,54 @@ function JoinChatroom({ chatroomId }: JoinChatroomProps) {
     }
   };
 
-  if (isChatroomLoading) {
-    return (
-      <div className="join-chatroom">
-        <div className="join-chatroom-loading">
-          <Loading />
-        </div>
-      </div>
-    );
-  }
-
-  if (!chatroom) {
-    return <NotFoundPage />;
-  }
-
   return (
-    <div className="join-chatroom">
-      <Header icon={IconVariant.ARROW_BACK} onClick={goBack}>
-        {`channel/${chatroom.name}`}
-      </Header>
-      <div className="join-chatroom-avatar">
-        <LargeAvatar
-          url={
+    <div className="join-chatroom-page">
+      <AvatarPageTemplate
```

Comment:
praise: Thanks for adding the template to these pages.

Reply:
Thank you!